test/components/plots/plotTests.ts

```diff
@@ -36,21 +36,21 @@ describe("Plots", () => {
 
       assert.strictEqual(1, r.renders, "initial render due to addDataset");
 
-      dFoo.broadcaster.broadcast();
+      (<any>dFoo)._dispatchChange();
```

Comment:
@aicioara You are correct in that we do use private fields in our test code and I do think that we should not disallow ourselves from doing so (There are valid arguments on both ends).  However, I think in this case, we can trigger our changes in a very direct way (use the `data` setter) and that if we can use the public API, then we should